cerberus/validator.py

```diff
@@ -666,8 +666,11 @@ def __normalize_sequence(self, field, mapping, schema):
             document_crumb=field, schema_crumb=(field, 'schema'),
             schema=schema)
         result = validator.normalized(document, always_return_document=True)
-        for i in result:
-            mapping[field][i] = result[i]
+        if len(result.values()) > 0:
+            if type(mapping[field]) is tuple:
```

Comment:
that wouldn't work with subclasses of `tuple`, therefore the `isinstance` function is common to identify types.
nonetheless, it should be enough to create a new object of a particular type: `mapping_field = type(mapping[field])(result.values)`